contrib/idxd-config-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+## Utility library for controlling and configuring Intel® In-Memory Analytics Accelerator (Intel® IAA) available on Next Generation Intel® Xeon® Scalable processors.
+set(LIBACCEL_SOURCE_DIR "${ClickHouse_SOURCE_DIR}/contrib/idxd-config")
```

Comment:
Suggest to put in similar platform restrictions (e.g. x86-only) like we have on top of contrib/qpl-cmake/CMakeLists.txt.

Ideally, you also add a comment to the restrictions in both files that they should be kept in sync.

Reply:
Done